### PR TITLE
[Blog] Open source journey @Layer5

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
This is my first ever blog on my open source journey at Layer5

**Notes for Reviewers**
Please read it thoroughly and suggest any suggestions like catchy headlines, subheading, or anything you like, I would like to hear your thoughts before publishing, This is the first time I have written a blog!!, :)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c4cec99e9dae8407b437ec2f2eb79adf466dc797 at: https://61d3e1debf0e3a62ab594296--layer5ng.netlify.app

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+</p>
+<h2 id="journey">How it started?</h2>
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Later5's slack workspace, I landed on different slack channels workspace, at first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks started to introduce themselves, my inner voice said when they can the why not me, so I also introduced my self to the community.
```
Comment (bariqhibat) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:32]: `... I landed on different slack channels workspace. At first`

Comment (bariqhibat) [reply src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:32]: `... so I also introduced myself to the community.`

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Later5's slack workspace, I landed on different slack channels workspace, at first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks started to introduce themselves, my inner voice said when they can the why not me, so I also introduced my self to the community. 
+The best part here in Layer5 is each and every newcomer is warmly welcomed and 
+they also have a dedicated  Newcomers Call which helped me to get started with the project and to know more about community works.
```
Comment (bariqhibat) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:34]: `... which helped them`

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+</p>
+<p>
+  I would highly recommend all the newcomers to engage as much as possible in the community, you will gradually start seeing a change in yourself.
+</p>
+
+<h3 id="first-pr">Way to my First Pull Request</h3>
+<p>
+I still remember at that time I was just a beginner with Git, HTML, CSS, and JavaScript and most of the issues on the repository were going over my head. 
+So I started with a very beginner-friendly issue and that was changing a background of a component and who would have thought this changing background issue will eventually change my life in open source.
```
Comment (bariqhibat) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:43]: `... beginner-friendly issue about changing ...`

**Review by bariqhibat — CHANGES_REQUESTED** [review]
Some grammatical comments

**Comment by Chadha93** [comment]
@bariqhibat thanks for the review and reading this, I'll update it.  😊

**Review by debo19 — COMMENTED** [review]
Suggested a few changes 😄

**Comment by Chadha93** [comment]
@bariqhibat @debo19 thanks for the reviews and suggestions I have updated the content accordingly.

**Comment by l5io** [comment]
🚀 Preview for commit 737ca6514a4ea91c26ce9695cccfe18fdde41d84 at: https://61d7fdd30ad76fc6640e3fe5--layer5ng.netlify.app

**Comment by debo19** [comment]
@Chadha93 Will you add a `description` field to the frontmatter? This is a short summary of the blog post. This text will show in the blog's list view and in community newsletters.

**Comment by Chadha93** [comment]
> @Chadha93 Will you add a `description` field to the frontmatter? This is a short summary of the blog post. This text will show in the blog's list view and in community newsletters.

okay, will add it.

**Comment by warunicorn19** [comment]
hey @Chadha93 could you please tie this off by tomorrow?

**Comment by Chadha93** [comment]
@debo19 @warunicorn19 updated this. Thanks for the reviews, you may add any other changes to this PR(if required).

**Comment by l5io** [comment]
🚀 Preview for commit 6adbd5297c67425e46d2ef3b6941cfe549afa852 at: https://61dbea751609f8588bfd53ba--layer5ng.netlify.app

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+title: My Open Source Journey @Layer5
+subtitle: It changed my life completely!
+date: 2022-01-03 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./profile-banner.png
+description: "My journey as an Open Source contributor at Layer5. How I started from scratch?, got involved in this community
```
Comment (warunicorn19) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:7]:
```suggestion
description: "My journey as an Open Source contributor at Layer5. How I started from scratch, got involved in this community
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Later5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice aksed, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part here in Layer5 is each and every newcomer is warmly welcomed and 
+they also have a dedicated  Newcomers Call which helped them to get started with the project and to know more about community works.
```
Comment (warunicorn19) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:36]:
```suggestion
they also have a dedicated  Newcomers Call which helped them to get started with the project and to know more about how the community works.
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Later5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice aksed, "If they can do it, why can't I?", so I went ahead and introduced myself to the community.
```
Comment (warunicorn19) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:34]:
```suggestion
As soon as I joined Layer5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx (hunk 5 above)
Comment (warunicorn19) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:36]:
```suggestion
they also have a dedicated  Newcomers Call which helped them to get started with the project and to know more about the community.
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@Chadha93, looking good. If you haven't already, you can include announcement of the start of your Layer5 internship in here. Seek out any opportunities to cross-link to other layer5.io pages and project sites.

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+import Profile from "./gaurav-chadha.png"
+
+<BlogWrapper>
+<img src={Profile} className="profileImg" align="right" alt="profile-img" />
+<p>
+  Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:28]:
```suggestion
My journey in open source started with late-night surfing on LinkedIn.
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:29]:
```suggestion
I saw a post about the Layer5 organization and open source which drove me to their website and [Slack workspace](https://slack.layer5.io). And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues. 
```

**Comment by l5io** [comment]
🚀 Preview for commit af2b2180633953af26add1ed6a52e2297a0dab39 at: https://61dc757eb1833408e9ff060f--layer5ng.netlify.app

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Layer5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community.
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:34]:
```suggestion
Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the [#newcomers channel](https://layer5io.slack.com/archives/C019426UBNY), but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Layer5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part here in Layer5 is each and every newcomer is warmly welcomed and
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:35]:
```suggestion
The best part for newcomers in Layer5 is that not only is each individual warmly welcomed, but that
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+<p>
+My journey in Open source started with late-night surfing on LinkedIn,
+I saw a post about Layer5 organization and open-source which drove me to their website and slack channel, and then Boom my one click to join @Layer5's slack workspace kickstarted my journey in Open Source.
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+As soon as I joined Layer5's slack workspace, I landed on different slack channels workspace, At first I was a bit shy to introduce myself on the #introduction channel but then I saw many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part here in Layer5 is each and every newcomer is warmly welcomed and 
+they also have a dedicated  Newcomers Call which helped them to get started with the project and to know more about how the community works.
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:36]:
```suggestion
Layer5 [MeshMates](/community/meshmates) and Community Managers host a weekly meeting (see the [community calendar](/events/calendar)) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of [Layer5's projects](/projects) and orients you to the how the community works (the [community handbook](/community/handbook) is an excellent resource in this regard) and who to contact in your areas of interest.
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+</p>
+<p>
+  I would highly recommend all the newcomers to engage as much as possible in the community, you will gradually start seeing a change in yourself.
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:39]:
```suggestion
  I highly recommend all newcomers to engage as much as possible in the community. You will gradually start seeing a change in yourself.
```

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+</p>
+
+<h3 id="first-pr">Way to my First Pull Request</h3>
+<p>
+I still remember at that time I was just a beginner with Git, HTML, CSS, and JavaScript and most of the issues on the repository were going over my head. 
+So I started with a very beginner-friendly issue about changing a background of a component and who would have thought this changing background issue will eventually change my life in open source.
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:45]: @Chadha93 include links to the help wanted and the good first issue labels. Make sure the links that you include span all orgs. See any of the project repos' READMEs for an example.

**Comment by leecalcote** [comment]
You might find a way to link to discuss.layer5.io as an alternate to Slack.

**Comment by leecalcote** [comment]
If you're looking for other graphic assets to include (you don't have to), here's one - https://camo.githubusercontent.com/26e03904afa984fce6265dd4d0b99b3effce7d159f51c3fa30757ce73dbac451/68747470733a2f2f646f63732e6d6573686572792e696f2f6173736574732f696d672f726561646d652f6c61796572352d636f6d6d756e6974792d7369676e2e706e67
- https://drive.google.com/open?id=1APabhklMj7h21DXRtNjgHfRS9bYK4zk3
- https://drive.google.com/open?id=19_cGcbtKbKdE9y6_6id891_MsDF1iF-x

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+author: Gaurav Chadha
+thumbnail: ./profile-banner.png
+description: "My journey as an Open Source contributor at Layer5. How I started from scratch, got involved in this community
+and gained many new experiences."
+category: Community
+tags:
+  - Open Source
+  - Experience
+  - Layer5
+  - Meshery
```
Comment (leecalcote) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:14]:
```suggestion
```

**Comment by l5io** [comment]
🚀 Preview for commit 8b7c5f7eac918b148f2b53a460299c2408a6fb2b at: https://61dcf44f2b3ea085d54415d4--layer5ng.netlify.app

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx (hunk 13 above)
Comment (Chadha93) [reply src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:44]: sounds good I'll add this

**Comment by Chadha93** [comment]
@leecalcote thanks for such great suggestions I'll update this, This will going be an amazing Blog 💪

**Comment by Chadha93** [comment]
@leecalcote I have updated it, you may have a look and I have changed the date of the blog to 15th Jan let's roll out this blog on 15th.. :)

**Comment by l5io** [comment]
🚀 Preview for commit e6eaaa0b2d4122847bc922c2ee032f3c4ae7b5e2 at: https://61e01e3218048a4107489e7a--layer5ng.netlify.app

File: src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx [hunk 15]
```diff
@@ -0,0 +1,81 @@
+---
+title: My Open Source Journey @Layer5
+subtitle: From Contributor to SWE Intern
+date: 2022-01-15 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./profile-banner.png
+description: "My journey as an Open Source contributor at Layer5. How I started from scratch, got involved in this community
+and gained many new experiences."
```
Comment (warunicorn19) [inline src/collections/blog/2022-03-01-open-source-journey-gaurav-chadha/index.mdx:8]:
```suggestion
and gained many new experiences."
type: Blog
```

**Comment by l5io** [comment]
🚀 Preview for commit 5abd64af57ef36267bba4c9121c579ae7a3487bf at: https://61e06b0653c29223e2ca31e5--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@Chadha93 could you please update the date on the folder name as well, let's publish today. (13th jan 2022)

**Comment by Chadha93** [comment]
> @Chadha93 could you please update the date on the folder name as well, let's publish today. (13th jan 2022)

Sure, I'll update the data on the folder, but let's publish it on 14Jan then.

**Comment by l5io** [comment]
🚀 Preview for commit 8165f199a0064bc18f15dc519dbf0f31d3d2ea69 at: https://61e129f78923afbe23e0fe69--layer5ng.netlify.app

**Comment by leecalcote** [comment]
uh-oh. each of the hyperlinked text have square bracket
Screen Shot 2022-01-14 at 1 23 24 PM
s around them.

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+
+<BlogWrapper>
+<img src={Profile} className="profileImg" align="right" alt="profile-img" />
+<p>
+  Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in open source started with late-night surfing on LinkedIn.
+I saw a post about the Layer5 organization and open source which drove me to their website and{" "}<a href = "https://slack.layer5.io">[Slack workspace]</a>. And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues.
```
Comment (warunicorn19) [inline src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:29]:
```suggestion
I saw a post about the Layer5 organization and open source which drove me to their website and{" "}<a href = "https://slack.layer5.io">Slack workspace</a>. And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues. 
```

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx [hunk 17]
```diff
@@ -0,0 +1,82 @@
...
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in open source started with late-night surfing on LinkedIn.
+I saw a post about the Layer5 organization and open source which drove me to their website and{" "}<a href = "https://slack.layer5.io">[Slack workspace]</a>. And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues. 
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">[#newcomers channel]</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community.
```
Comment (warunicorn19) [inline src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:34]:
```suggestion
Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers channel</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
```

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">[#newcomers channel]</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part for newcomers in Layer5 is that not only is each individual warmly welcomed, but that
+Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">[community calendar]</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">[Layer5's projects]</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">[community handbook]</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```
Comment (warunicorn19) [inline src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:36]:
```suggestion
Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">[community calendar]</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Merge at-will, @warunicorn19

**Comment by l5io** [comment]
🚀 Preview for commit 317689a473fb5eace70ae01ca014a51328de2fda at: https://61e1d4cc2224af109ffda9f7--layer5ng.netlify.app

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+<p>
+Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers channel</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part for newcomers in Layer5 is that not only is each individual warmly welcomed, but that
+Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">[community calendar]</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```
Comment (warunicorn19) [inline src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:36]:
```suggestion
Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">community calendar</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
```

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx [hunk 20]
```diff
@@ -0,0 +1,82 @@
...
+</p>
+
+<h3 id="first-pr">Way to my First Pull Request 🏁</h3>
+<p>
+I still remember at that time I was just a beginner with Git, HTML, CSS, and JavaScript and most of the issues on the repository were going over my head. 
+So I started with a very beginner-friendly issue about changing a background of a component and who would have thought this changing background issue will eventually change my life in open source. 
+Then I made my first ever pull request in Layer5 and that got merged!
+</p>
+<p>
+Try out some good first issues here{""} <a href = "https://github.com/layer5io/layer5/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">[Layer5]</a> {""} <a href = "https://github.com/meshery/meshery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">[Meshery]</a>💻
```
Comment (warunicorn19) [inline src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:49]:
```suggestion
Try out some good first issues here{""} <a href = "https://github.com/layer5io/layer5/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">Layer5</a> {""} ,<a href = "https://github.com/meshery/meshery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">Meshery</a>💻
```

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx (hunk 19 above)
Comment (leecalcote) [reply src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx:36]: Actually... this should be a `<Link>`, yeah?

File: src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx (hunk 20 above)
Comment (leecalcote) [reply src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx]: `<Link>` for intra-site hyperlinks. `<a>` for external hyperlinks

**Pull request merged**